src/Identity.Governance/Identity.Governance/custom/Get-MgEntitlementManagementAccessPackage.ps1

```diff
@@ -0,0 +1,210 @@
+    [System.Int32]
+    # Show only the first n items
+    ${Top},
+
+    [Parameter(DontShow)]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # Wait for .NET debugger to attach
+    ${Break},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [Microsoft.Graph.PowerShell.Runtime.SendAsyncStep[]]
+    # SendAsync Pipeline Steps to be appended to the front of the pipeline
+    ${HttpPipelineAppend},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [Microsoft.Graph.PowerShell.Runtime.SendAsyncStep[]]
+    # SendAsync Pipeline Steps to be prepended to the front of the pipeline
+    ${HttpPipelinePrepend},
+
+    [Parameter(DontShow)]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Uri]
+    # The URI for the proxy server to use
+    ${Proxy},
+
+    [Parameter(DontShow)]
+    [ValidateNotNull()]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.PSCredential]
+    # Credentials for a proxy server to use for the remote call
+    ${ProxyCredential},
+
+    [Parameter(DontShow)]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # Use the default credentials for the proxy
+    ${ProxyUseDefaultCredentials},
+
+    [Parameter(ParameterSetName='ListAll')]
+    [Parameter(ParameterSetName='DisplayNameEq')]
+    [Parameter(ParameterSetName='DisplayNameContains')]
+    [Parameter(ParameterSetName='CatalogId')]
+    [Microsoft.Graph.PowerShell.Category('Runtime')]
+    [System.Management.Automation.SwitchParameter]
+    # List all pages.
+    ${All}
+)
+
+begin {
+    try {
+        $outBuffer = $null
+        if ($PSBoundParameters.TryGetValue('OutBuffer', [ref]$outBuffer)) {
+            $PSBoundParameters['OutBuffer'] = 1
+        }
+        $parameterSet = $PSCmdlet.ParameterSetName
+
+        if ($parameterSet -eq "DisplayNameEq") {
+            
+            $Filter = "displayName eq '{0}'" -f $DisplayNameEq
+            $PSBoundParameters['Filter'] = $Filter
+            $null = $PSBoundParameters.Remove('DisplayNameEq')
+        
+        } elseif ($parameterSet -eq "DisplayNameContains") {
+            
+            $Filter = "contains(tolower(displayName), '{0}')" -f $DisplayNameContains
+            $PSBoundParameters['Filter'] = $Filter
+            $null = $PSBoundParameters.Remove('DisplayNameContains')
+        } elseif ($parameterSet -eq "CatalogId") {
+
+            $Filter = "accessPackageCatalog/Id eq '{0}'" -f $CatalogId
+            $PSBoundParameters['Filter'] = $Filter
+            $null = $PSBoundParameters.Remove('CatalogId')
+        }
+        
+        if ($PSBoundParameters.ContainsKey('Top') -or $PSBoundParameters.ContainsKey('All')) {
+
+        } else {
+            $PSBoundParameters['All'] = $true
+        }
```

Comment:
We should then move everything in the begin block to the process block once [this suggestion](https://github.com/microsoftgraph/msgraph-sdk-powershell/pull/632/files#r627848030) is applied. See the example [here](https://github.com/Azure/autorest.powershell/blob/master/docs/customization.md#adding-a-single-variant) and [here](https://github.com/Azure/azure-powershell/blob/generation/src/Dns/custom/Get-AzDnsResourceReference.ps1#L82-L88).

Reply:
Moved into process block across this PR.